lib/ansible/module_utils/service.py

```diff
@@ -114,7 +114,7 @@ def fail_if_missing(module, found, service, msg=''):
     :kw msg: extra info to append to error/success msg when missing
     '''
     if not found:
-        if module.check_mode:
+        if module.check_mode and not fail_in_check_mode:
```

Comment:
i would just remove this part and always fail when not present

Reply:
@bcoca , will do, thanks!

Reply:
changed.
FreeBSD is handled differently in `service` module. I'm not sure it's worth touching that